modules/apps/transfer/keeper/relay_test.go

```diff
@@ -913,3 +913,87 @@ func (suite *KeeperTestSuite) TestOnTimeoutPacketSetsTotalEscrowAmountForSourceI
 	totalEscrowChainB = suite.chainB.GetSimApp().TransferKeeper.GetTotalEscrowForDenom(suite.chainB.GetContext(), coin.GetDenom())
 	suite.Require().Equal(sdkmath.ZeroInt(), totalEscrowChainB.Amount)
 }
+
+func (suite *KeeperTestSuite) TestPacketBackwardsCompatibility() {
+	// We are testing a scenario where a packet in the future has a new populated
+	// field called "new_field". And this packet is being sent to this module which
+	// doesn't have this field in the packet data. The module should be able to handle
+	// this packet without any issues.
+
+	var packetData []byte
+
+	testCases := []struct {
+		msg      string
+		malleate func()
+		expPass  bool
+	}{
+		{
+			"success: new field",
+			func() {
+				jsonString := fmt.Sprintf(`{"denom":"denom","amount":"100","sender":"%s","receiver":"%s","memo":"memo","new_field":"value"}`, suite.chainB.SenderAccount.GetAddress().String(), suite.chainA.SenderAccount.GetAddress().String())
+				packetData = []byte(jsonString)
+			},
+			true,
```

Comment:
since we're adding this as a new test maybe we can use the `expError` here instead?